src/pybind/mgr/mgr_module.py

```diff
@@ -44,7 +45,12 @@
     "snaptrim_wait",
     "snaptrim_error",
     "creating",
-    "unknown"]
+    "unknown",
+    "premerge",
+    "failed_repair",
+    "laggy",
+    "wait",
```

Comment:
These two states are not relevant for nautilus. You can look at the complete list of states in https://github.com/ceph/ceph/blob/nautilus/src/osd/osd_types.cc#L1143-L1213

Reply:
Thank you for the hint, @neha-ojha! :) Will update the PR accordingly.

Reply:
Updated the PR and the commit message accordingly!